invenio_rdm_records/jsonschemas/records/record-v1.0.0.json

```diff
@@ -4,61 +4,291 @@
   "title": "Invenio Datacite based Record Schema v1.0.0",
   "type": "object",
   "additionalProperties": false,
+
   "properties": {
-    "_access": {
-      "metadata_restricted": {
-        "default": false,
-        "description": "Record metadata accesibility. Public by default (False).",
-        "type": "boolean"
-      },
-      "files_restricted": {
-        "default": false,
-        "description": "Record associated files accesibility. Public by default (False).",
-        "type": "boolean"
-      }
-    },
     "_bucket": {
       "description": "Record bucket.",
       "type": "string"
     },
-    "access_right": {
-      "default": "open",
-      "description": "Access right for record.",
+    "_recid": {
+      "description": "Invenio record identifier (alphanumeric).",
+      "type": "string"
+    },
+    "_conceptrecid": {
+      "description": "Invenio record identifier (integer).",
       "type": "string"
     },
-    "additional_descriptions": {
+    "_visibility": {
+      "description": "Visibility of the record (CV).",
+      "type": "string"
+    },
+    "_visibility_files": {
+      "description": "Visibility of the files of a record (CV).",
+      "type": "string"
+    },
+    "_access_condition": {
+      "type": "object",
+      "properties": {
+        "condition": {"type": "string"},
+        "link_validity": {"type": "string"}
```

Comment:
> Tests for access_condition since the access part is not fully defined yet. It is not added to the fixtures either.

Sorry I was not really clear on my comment above. I am not fully sure. I think it depends on how we do share. I did not implement tests or anything for this field. I think it will change in the future with access. Shall I remove it? WDYT?

Reply:
Yes, let's remove it for now. Your yoda quote is a motto: let's have things that work and are clear committed. If things are halfway, we can get to them later. A reason for that, is it will help us trust the written code, so we can base further functionality on it, rather than always wander if it was a temporary half-done thing.